HopsanGUI/HcomHandler.cpp

```diff
@@ -381,6 +381,21 @@ HcomHandler::HcomHandler(TerminalConsole *pConsole) : QObject(pConsole)
     registerInternalFunction("x", "Returns the X-vector of the specified variable.","Usage: x(vector)");
     registerInternalFunction("td", "Converts variable to time domain." "Usage: y = td(x)");
     registerInternalFunction("fd", "Converts variable to frequency domain." "Usage: y = fd(x)");
+    registerInternalFunction("expr", "Defines a new SymHop expression" "Usage: e = expr(...)");
```

Comment:
Maybe call it something else then "SymHop" expression. It is not clear to the end-user what SymHop is. Maybe call it mathematical expression or something similar. Also in all other places, and related documentation. 
I did this mistake whit NumHop, and I intend to replace all mention of NumHop with "Embedded script" since it makes more sense to the end users. We should do the same with SymHop. NumHop and SymHop are only the name of the libraries implementing the functionallity.

Reply:
Good point. It is obvious to me what it means, but not the end user. I changed it to "symbolic expression".